extensions/gitlab/src/mrmenu.tsx

```diff
@@ -87,7 +88,10 @@ export default function MenuCommand(): JSX.Element {
           >
             {mrsReview?.map((m) => (
               <MenuBarItem
-                icon={"mropen.png"}
+                icon={{
+                  source: GitLabIcons.merge_request,
+                  tintColor: { light: "#000", dark: "#FFF", adjustContrast: false },
```

Comment:
I adjusted the `tintColor` to HEX values and set `adjustContrast` to `false` here because the `Color.PrimaryText` appeared slightly different from the white text, and I wanted to maintain a cohesive look throughout.